scripts/freebsd/update.sh

```diff
@@ -19,8 +19,10 @@ fi
 
 # Update FreeBSD
 # NOTE: this will fail if there aren't any patches available for the release yet
-env PAGER=/bin/cat $freebsd_update fetch;
-env PAGER=/bin/cat $freebsd_update install;
+if [ "$major_version" -lt 11 ]; then
+  env PAGER=/bin/cat $freebsd_update fetch;
+  env PAGER=/bin/cat $freebsd_update install;
+fi
```

Comment:
No this was just a quick work around that update fails when there are no patches to install. I have done a proper solution for [boxcutter/bsd](https://github.com/boxcutter/bsd/blob/master/script/freebsd/update.sh#L22). I'll port that. 

Thanks for reminding me.